### DISCUSSION TITLE
How To Submit A PR To homebrew-cask-versions

### DISCUSSION THREAD

**Discussion by tmrdev** [Homebrew/discussions#4907] [Writing Formulae/Casks]
### Output of `brew config`

```shell
brew config
HOMEBREW_VERSION: 4.1.20
ORIGIN: https://github.com/Homebrew/brew
HEAD: d3d51d3448966b5250b5c60110c945040054803d
Last commit: 4 days ago
Core tap origin: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 1937ed79bc5984539efc7f666d5fffaf9a20f6a8
Core tap last commit: 12 days ago
Core tap branch: master
Core tap JSON: 15 Nov 22:33 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_DISPLAY: /private/tmp/com.apple.launchd.tquJ4keoFI/org.xquartz:0
HOMEBREW_MAKE_JOBS: 10
HOMEBREW_SORBET_RUNTIME: set
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: 10-core 64-bit arm_blizzard_avalanche
Clang: 15.0.0 build 1500
Git: 2.39.3 => /Applications/Xcode.app/Contents/Developer/usr/bin/git
Curl: 8.1.2 => /usr/bin/curl
macOS: 14.0-arm64
CLT: 15.0.0.0.1.1694021235
Xcode: 15.0
Rosetta 2: false
```

### Output of `brew doctor`

```shell
brew doctor
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: You have uncommitted modifications to Homebrew/homebrew-cask.
If this is a surprise to you, then you should stash these modifications.
Stashing returns Homebrew to a pristine state but can be undone
should you later need to do so for some reason.
  cd /opt/homebrew/Library/Taps/homebrew/homebrew-cask && git stash -u && git clean -d -f

Uncommitted files:
  ?? Casks/g/greenery-beta.rb
```

### Description of issue

I'm trying to submit through a PR the Cask for the Greenery Beta version (v.0.9.3) to the Beta/Unstable branch at homebrew/homebrew-cask-versions

It should be noted that this Greenery Beta is a Binary release only, so there is no source code for the app.

I installed homebrew-cask-versions with this command and then used cd to enter the directory:

  $: brew tap homebrew/cask-versions

Then I used the following command to create the Cask:

  $: brew create --cask https://github.com/GreenfireInc/Releases.Greenery/releases/download/v0.9.3/Greenery.zip --set-name greenery-beta
  
After saving the Cask file (greenery-beta.rb) I noticed that it is in homebrew/homebrew-cask. Full path below:

  /opt/homebrew/Library/Taps/homebrew/homebrew-cask
  
Is this the proper method for submitting a PR for a Binary Beta Release or should greenery-beta.rb be created in the homebrew/homebrew-cask-versions directory?  The brew create command sets where the file is saved so am I issuing the command correctly?

**Comment by SMillerDev** [comment]
You can just move the file, although `brew create` might also have a `--tap` option
